### ISSUE TITLE
Punkte für Zitate

### ISSUE THREAD

**Issue by MatthiasApsel** [ckruse/cforum#494]
siehe https://forum.selfhtml.org/meta/2015/nov/4/zitatesammlung-und-punkte/1653672#m1653672 ff.
Vielleicht haben noch andere eine Meinung dazu.

**Comment by MatthiasApsel** [comment]
Es könnte ebenso einen Punkt für den Zitatautor geben, wenn das Zitat in einem Beitrag verlinkt wird.

**Comment by campingrider** [comment]
Ich möchte das noch einmal unterstreichen - mMn ist es schon punktwürdig, wenn man eine Aussage verfasst, die es würdig ist, in die Zitatesammlung aufgenommen zu werden.

Ich würde allerdings davon absehen, dem Vorschlagenden Punkte zu geben, das halte ich für unnötig (und unbegründet).

Ich könnte mir also gut vorstellen, dass der Autor eines Zitats, sagen wir mal, 5 Punkte erhält **sobald das Zitat die Abstimmung mit positivem Ausgang überstanden hat**. Statt 5 Punkten könnte ich mir hier auch 10, 15 oder sogar 20 Punkte vorstellen - seit die Zitatesammlung mit den aktuellen Änderungen mehr ins Auge der Öffentlichkeit gerückt wurde bedeutet eine Aufnahme in die Zitatesammlung so gut wie immer, dass sehr viele Nutzer positiv für dieses Zitat abgestimmt haben - daher fände ich auch 20 Punkte für den Zitatautor angemessen (es ist eine größere Leistung, eine zitierwürdige Aussage zu treffen, als nur eine, die einem bestimmten Benutzer gefällt, was 10 Punkten entspricht).

Wenn der Vorschlagende auch Punkte bekommen soll, dann deutlich weniger als der Autor, lediglich in Anerkennung der Ergreifung der Initiative - irgendwas zwischen 1 und 5 Punkte.

**Comment by auge8472** [comment]
> Ich könnte mir also gut vorstellen, dass der Autor eines Zitats, sagen wir mal, 5 Punkte erhält sobald das Zitat die Abstimmung mit positivem Ausgang überstanden hat. Statt 5 Punkten könnte ich mir hier auch 10, 15 oder sogar 20 Punkte vorstellen [weil …]

@campingrider: Dass der Zitierte nach der Übernahme des Zitats Punkte erhält, halte ich für eine gute Idee. Es sollte sich dabei aber nicht um 15 oder gar 20 Punkte handeln, auch wenn ich deine Argumentation nachvollziehen kann. Für ein akzeptiertes Posting gibt es „nur“ 15 Punkte. Es würde sich *zumindest für mich* komisch anfühlen, wenn der Zitatautor „nur“ für das Zitat mehr Punkte bekommt als der Autor einer akzeptierten Antwort für ein ganzes Posting.

Fünf oder zehn Punkte würde ich dem Autor aber ohne weiteres zugestehen wollen. Eine Perle ist nunmal eine Perle und viele der Zitate sind genau dies.

> Es könnte ebenso einen Punkt für den Zitatautor geben, wenn das Zitat in einem Beitrag verlinkt wird.

Das halte ich für zuviel des Guten. Ja, ein weiterhin propagiertes, weil verlinktes Zitat ist „mehr wert“ als eines, das auf ewig unbeachtet im Stapel liegt. Andererseits würde man sich – ohne entsprechende Vorkehrungen – selbst Punkt um Punkt zuschustern können.

Z.B. Gunnar, der sich ja selbst oft genug selbst zitiert, würde davon, wenn auch, davon gehe ich definitiv aus, ungewollt, davon profitieren.  Also *das* sein lassen oder Vorkehrungen treffen.

**Comment by MatthiasApsel** [comment]
Ja, [meine Punktvorschläge](https://forum.selfhtml.org/meta/2015/nov/4/zitatesammlung-und-punkte/1653700#m1653700) sind zuviel des Guten. Als ich die unterbreitete, bekamen die Vorschläge mal 3 oder 4 Punkte. Heute sind es mal gleich 12. Deshalb schlage ich vor:

ausschließlich wenn ein Zitat übernommen wird
- bekommt der Autor die Summe der Ja-Stimmen, mindestens jedoch 10 Punkte 
- bekommt der Vorschlagende die Summe der Ja-Stimmen, höchstens jedoch 5 Punkte
- jede Erwähnung des Zitats durch einen anderen als den Autor bringt dem Autor einen Punkt

**Comment by campingrider** [comment]
> * bekommt der Autor die Summe der Ja-Stimmen, mindestens jedoch 10 Punkte
> * bekommt der Vorschlagende die Summe der Ja-Stimmen, höchstens jedoch 5 Punkte

Damit wäre ich einverstanden.

> * jede Erwähnung des Zitats durch einen anderen als den Autor bringt dem Autor einen Punkt

Das fände ich akzeptabel - allerdings nicht unbedingt nötig. Diese Regel ist mMn trotz der Klausel anfällig für unerwünschte Nebeneffekte (wenn ich auch die Argumentation gut nachvollziehen kann). Mindestens müssen auch Signaturen ausgeschlossen werden, dann kann ich wahrscheinlich gut damit leben.

**Comment by MatthiasApsel** [comment]
Wenn einen Wunsch frei hätte, würde ich die Umsetzung dieser Issue vorantreiben #Drängelposting ;-)
